### DISCUSSION TITLE
Não consigo criar um novo repositório!

### DISCUSSION THREAD

**Discussion by Eudes89** [community/community#59026] [Repositories] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Ao digitar o nome do repositório ele simplesmente da um erro dizendo : algo deu errado. 
Mais alguém está passando por esse problema?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @Eudes89 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Reply by Eudes89** [reply]
Não ter moderadores para o português eu até entendo, mas fechar a pergunta por esse motivo mostra uma falta de respeito enorme com a dúvida em questão e com as pessoas que não sabem o inglês. Se o moderador desejar ver o que está escrito existem inúmeros programas e sites que fazem essa tradução.
Desculpem pelo texto que não condiz com uma solução para a dúvida.

**Comment by DevTanzin** [comment]
I'm having trouble creating a new repository when you name the repository it says something is wrong I've been having this problem for a long time, can anyone help me? Is anyone else having this same problem?

**Reply by AgnaldoVaz** [reply]
I'm also having the same problem.

**Reply by Eudes89** [reply]
Aparentemente o meu problema foi resolvido com o tempo, isso mostra que era algum defeito interno do próprio Git. Espero que esse problema se resolva para vocês.

**Reply by Kemadu** [reply, marked as answer]
Também tive o mesmo problema mas eu consegui resolver desativando o google tradutor do GitHub
I also had the same problem, but I managed to solve it by disabling Google Translate on GitHub

**Reply by Eudes89** [reply]
Eu tive ess problema com IA e resolvi da mesma forma, desativando o tradutor automático.
Talvez essa seja a solução mais viável.
Obrigado @Kemadu
